src/restconfig/src/main/java/org/geoserver/rest/catalog/NamespaceController.java

```diff
@@ -114,6 +94,7 @@ public RestWrapper getNamespaces() {
         if ( catalog.getWorkspaceByName( namespace.getPrefix() ) == null ) {
             WorkspaceInfo ws = catalog.getFactory().createWorkspace();
             ws.setName( namespace.getPrefix() );
+            ws.setIsolated(namespace.isIsolated());
```

Comment:
The name of the method thru me off balance, was wondering what was going on here... namespaceGet, but it's actually a namespacePost :-)
I'd fix the method name, but I understand it's not your fault for it being there. @tbarsballe git blame points at you, probably some mass refactor side effect by the commit message.

Reply:
Yeah, that commit was most likely part of a mass refactor, probably moving the module from rest-ng to rest.
As for the method name, I assume it was some copy-paste that got missed during the code sprint. I've found and fixed at least one other similar issue elsewhere in the REST api. At least the MVC annotations make that sort of issue a bit easier to spot.